lib/bolognese/readers/codemeta_reader.rb

```diff
@@ -58,12 +60,14 @@ def read_codemeta(string: nil, **options)
           { "subject" => s }
         end
 
+        titles =  meta.fetch("title", nil).nil? ?  [{ "title" => meta.fetch("name", nil) }] : [{ "title" => meta.fetch("title", nil) }]  
```

Comment:
Again, I don't understand this use of the terniary operator.

Reply:
I think `title` doesn't exist in codemeta-v2 (https://raw.githubusercontent.com/codemeta/codemeta/2.0/codemeta.jsonld) but it does in codemeta-v1  (https://raw.githubusercontent.com/codemeta/codemeta/1.0/codemeta.jsonld). I wanted to account for reading metadata in both versions.